lib/SIL/SILVerifier.cpp

```diff
@@ -577,6 +578,38 @@ class SILVerifier : public SILVerifierBase<SILVerifier> {
     SILLocation::LocationKind LocKind = L.getKind();
     ValueKind InstKind = I->getKind();
 
+    // Check that there is at most one debug variable defined
+    // for each argument slot.
+    if (!DS->InlinedCallSite) {
+      SILDebugVariable VarInfo;
+      if (auto *DI = dyn_cast<AllocStackInst>(I)) {
```

Comment:
Is this idiomatic, or are we typically doing a switch (I->getKind()) followed by cast<>s?